targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L475xG/device/TOOLCHAIN_IAR/stm32l475xx.icf

```diff
@@ -33,7 +33,7 @@ if (!isdefinedsymbol(MBED_BOOT_STACK_SIZE)) {
 }
 
 define symbol __size_cstack__ = MBED_BOOT_STACK_SIZE;
-define symbol __size_heap__   = 0x17000;
+define symbol __size_heap__   = 0x13C00;
 define block CSTACK    with alignment = 8, size = __size_cstack__   { };
 define block HEAP      with alignment = 8, size = __size_heap__     { };
```

Comment:
```suggestion
define block HEAP      with expanding size, minimum size = __size_heap__, alignment = 8     { };
```
Dynamic HEAP for IAR  @juhoeskeli - Please verify this with IAR 8.x

Reply:
Deepika's suggestion is good. We don't waste RAM on a already constrained platform in builds without LWIP.

Reply:
Thanks @deepikabhavnani . I will verify this and report back results.

Reply:
Seems to work fine. Will do commit with the suggested change.

without PPP and with expanding change:

"P2", part 2 of 2:                             0x17820
  HEAP                            0x2000'07e0  0x17820  <Block>
    HEAP                 uninit   0x2000'07e0  0x17820  <Block tail>
                                - 0x2001'8000  0x17820

with PPP and with expanding change:
"P2", part 3 of 3:                             0x144e8
  HEAP                            0x2000'3b18  0x144e8  <Block>
    HEAP                 uninit   0x2000'3b18  0x144e8  <Block tail>
                                - 0x2001'8000  0x144e8


with&without PPP and without expanding change:
"P2", part 3 of 3:                             0x13c00
  HEAP                            0x2000'3b18  0x13c00  <Block>
    HEAP                 uninit   0x2000'3b18  0x13c00  <Block tail>
                                - 0x2001'7718  0x13c00